src/java/boa/datagen/DefaultProperties.java

```diff
@@ -74,6 +74,11 @@
 
 	public static String localDataPath = null;
 	
+	// for ML
+	public static String HADOOP_SEQ_FILE_LOCATION = "";
+	public static String HADOOP_OUT_LOCATION = "";
+	//public static String HADOOP_OUT_LOCATION = "/home/tess/Desktop/datagen";
```

Comment:
Are these changes necessary?

Reply:
Current, it is used to store the trained model.

Reply:
Perhaps the commented out code on line 80 can be removed.

Reply:
Yes, I have removed this line.